app/assets/javascripts/controllers/alerts/alerts_overview_controller.js

```diff
@@ -22,7 +21,7 @@ angular.module('ManageIQ').controller('alertsOverviewController',
       }
 
       function setupConfig() {
-        vm.category = alertsCenterService.categories[0];
+        vm.category = alertsCenterService.categories[0].value;
 
         vm.unGroupedGroup =         {
           value: __('Ungrouped'),
```

Comment:
this should probably be

```
value: 'Ungrouped',
title: N_('Ungrouped'),
```

since value should not be translated, and title would get translated twice, thanks to the `translate` function